packages/webapp/src/containers/Consent/locales/en/Owner.Consent.md

```diff
@@ -1,30 +1,47 @@
 ##### Version 3.0:
 
```

Comment:
This seems to be the only file that is missing a `&nbsp;` here. Make this consistent over all files.

Reply:
are you sure? I'm seeing the &nbsp there